src/Content/Text/BBCode.php

```diff
@@ -1473,10 +1471,6 @@ public static function convert($text, $preserve_nl = false, $try_oembed = true,
 
 		$text = str_replace(["\r","\n"], ['<br />', '<br />'], $text);
 
-		if ($preserve_nl) {
```

Comment:
Full review will have to wait until evening, but one question in advance: Wasn't the parameter in use?

Reply:
Only in the `impressum` addon, but even then, the string replacement was ineffective because the same replacement was performed just a couple lines before.

Reply:
Yeah, I see. Line 1474 already does some job that makes the job for line 1477 obsolete.